src/rgw/rgw_admin.cc

```diff
@@ -157,6 +158,9 @@ void _usage()
   cout << "  orphans find               init and run search for leaked rados objects (use job-id, pool)\n";
   cout << "  orphans finish             clean up search for leaked rados objects\n";
   cout << "  orphans list-jobs          list the current job-ids for orphans search\n";
+  cout << "  role create                create a AWS role for use with STS \n";
+  cout << "  role delete                delete a role\n";
+  cout << "  role get                   get a role\n";
```

Comment:
nit: extra space after `STS`

also, there's a unit test that compares the output of `radosgw-admin -h` against `test/cli/radosgw-admin/help.t` - could you please update that file to match your changes? jenkins is complaining about it:
```
/home/jenkins-build/build/workspace/ceph-pull-requests/src/test/cli/radosgw-admin/help.t: failed
--- /home/jenkins-build/build/workspace/ceph-pull-requests/src/test/cli/radosgw-admin/help.t
+++ /home/jenkins-build/build/workspace/ceph-pull-requests/src/test/cli/radosgw-admin/help.t.err
@@ -108,6 +108,9 @@
     orphans find               init and run search for leaked rados objects (use job-id, pool)
     orphans finish             clean up search for leaked rados objects
     orphans list-jobs          list the current job-ids for orphans search
+    role create                create a AWS role for use with STS 
+    role delete                delete a role
+    role get                   get a role
   options:
```